### ISSUE TITLE
Dont fail if unrelated test case is missing

### ISSUE THREAD

**Issue by rjfarmer** [MESAHub/mesa_test#25]
b6f46d4 failed to run any star/ test case as dev_pre_ms_to_cc_140 was listed in do1_test_source but did not exist in star/test_suite. So if a test case is missing it should be flagged somehow? but probably should not break running any other test case.

**Comment by rjfarmer** [comment]
````
1.3M_ms_high_Z 1
Email, password, and computer name accepted
/home/rfarmer/lib/ruby/gems/2.5.0/gems/mesa_test-1.0.0/lib/mesa_test.rb:851:in `check_test_case': No such test case: /home/rfarmer/data/mesa/mesatest_git/tmp.yzaIYSlK21
/star/test_suite/dev_pre_ms_to_cc_140. (TestCaseDirError)
	from /home/rfarmer/lib/ruby/gems/2.5.0/gems/mesa_test-1.0.0/lib/mesa_test.rb:809:in `initialize'
	from /home/rfarmer/lib/ruby/gems/2.5.0/gems/mesa_test-1.0.0/lib/mesa_test.rb:605:in `new'
	from /home/rfarmer/lib/ruby/gems/2.5.0/gems/mesa_test-1.0.0/lib/mesa_test.rb:605:in `block (2 levels) in load_test_source_data'
	from /home/rfarmer/lib/ruby/gems/2.5.0/gems/mesa_test-1.0.0/lib/mesa_test.rb:601:in `each'
	from /home/rfarmer/lib/ruby/gems/2.5.0/gems/mesa_test-1.0.0/lib/mesa_test.rb:601:in `block in load_test_source_data'
	from /home/rfarmer/lib/ruby/gems/2.5.0/gems/mesa_test-1.0.0/lib/mesa_test.rb:889:in `visit_dir'
	from /home/rfarmer/lib/ruby/gems/2.5.0/gems/mesa_test-1.0.0/lib/mesa_test.rb:600:in `load_test_source_data'
	from /home/rfarmer/lib/ruby/gems/2.5.0/gems/mesa_test-1.0.0/bin/mesa_test:364:in `update_mesa'
	from /home/rfarmer/lib/ruby/gems/2.5.0/gems/mesa_test-1.0.0/bin/mesa_test:353:in `create_and_check_mesa'
	from /home/rfarmer/lib/ruby/gems/2.5.0/gems/mesa_test-1.0.0/bin/mesa_test:50:in `test'
	from /home/rfarmer/lib/ruby/gems/2.5.0/gems/thor-0.20.3/lib/thor/command.rb:27:in `run'
	from /home/rfarmer/lib/ruby/gems/2.5.0/gems/thor-0.20.3/lib/thor/invocation.rb:126:in `invoke_command'
	from /home/rfarmer/lib/ruby/gems/2.5.0/gems/thor-0.20.3/lib/thor.rb:387:in `dispatch'
	from /home/rfarmer/lib/ruby/gems/2.5.0/gems/thor-0.20.3/lib/thor/base.rb:466:in `start'
	from /home/rfarmer/lib/ruby/gems/2.5.0/gems/mesa_test-1.0.0/bin/mesa_test:386:in `<top (required)>'
	from /home/rfarmer/bin/mesa_test:23:in `load'
	from /home/rfarmer/bin/mesa_test:23:in `<main>'
````

**Comment by wmwolf** [comment]
I'll see what I can do to make `mesa_test` ignore missing test cases and just move on. Note that `each_test_run` has an exit statement that I believe will get triggered if it can't `cd` to a test case, so while I may be able to solve the issue for an individual call to `mesa_test test TEST_NAME`, I think the problem just gets kicked to mesa for a call that triggers a vanilla run of `each_test_run`, like `mesa_test install_and_test main`, which outsources the running of all test cases to that script.

**Comment by jschwab** [comment]
We can easily make the each_test_* scripts `return` instead of `exit` if you'd prefer that they keep going.

**Comment by wmwolf** [comment]
Yes, I think making that change is best. The fix is live in `mesa_test` v1.0.5, and it seems to be working well on the aforementioned commit, so closing this now.

**Comment by jschwab** [comment]
Okay. I just made that change on `main`.
